### PR TITLE
Implemented data transfer from firebase

### PR DESCRIPTION
**Pull request by KasperLeiszner**
Solves issue #27
* Rewrote route in app.js
* Modified template project-overview
* Added weeks getter in project model
* Implemented and Exported new funcktion getAllProjects
* Added week colorstyles in stylesheet

### CODE REVIEW

File: src/controllers/controller.js [hunk 1]
```diff
@@ -50,13 +50,34 @@ function addDeveloperToProject(project, developer) {
 
 //TODO deleteDeveloperFromProject
 
+async function getAllProjects() {
+  let projectSnapshot = await db.collection('Projects').get();
+  let projectArray = [];
```
Comment (jhviggo) [inline src/controllers/controller.js:55]: both const

File: src/controllers/controller.js [hunk 2]
```diff
@@ -50,13 +50,34 @@ function addDeveloperToProject(project, developer) {
...
+
+  for (project of projectSnapshot._docs()) {
+    let projectData = project.data();
+    let projectObj = new Project(project.id, projectData.name, new Date(projectData.startDate.toDate()), new Date(projectData.endDate.toDate()), projectData.estimatedHours);
```
Comment (jhviggo) [inline src/controllers/controller.js:59]: more consts. Also what's this `toDate()` method?

File: src/controllers/controller.js [hunk 3]
```diff
@@ -50,13 +50,34 @@ function addDeveloperToProject(project, developer) {
...
+    projectArray.push(projectObj);
+    
+    let developerInProjectSnapshot = await db.collection('DevelopersInProject').where('projectDocId', '==', projectObj.docId).get();
```
Comment (jhviggo) [inline src/controllers/controller.js:62]: more const. All the one time things can be const :stuck_out_tongue:

File: test/projectTest.spec.js [hunk 4]
```diff
@@ -112,7 +112,7 @@ describe('developer', () => {
         proj1.addDeveloper(dev1);
         proj1.addDeveloper(dev2);
         proj1.addDeveloper(dev3);
-        assert.isTrue(proj1.weekColorCode(20) === 'GREEN');
+        assert.isTrue(proj1.weekColorCode(20) === 'week-green');
```
Comment (jhviggo) [inline test/projectTest.spec.js:115]: For these asserts i would probably use `.equal(a, b)`

File: src/controllers/controller.js (hunk 2 above)
Comment (KasperLeiszner) [reply src/controllers/controller.js:59]: Firebase giver timestamps og ikke dates. Med toDate bliver det converteret til en date

File: test/projectTest.spec.js (hunk 4 above)
Comment (KasperLeiszner) [reply test/projectTest.spec.js:115]: Har ikke skrevet testen

Comment (jhviggo) [reply test/projectTest.spec.js:115]: Ah i see

**Pull request merged**